modules/local/samplesheet_check.nf

```diff
@@ -14,15 +14,18 @@ process SAMPLESHEET_CHECK {
     path '*.csv'       , emit: csv
     path "versions.yml", emit: versions
 
-    script: // This script is bundled with the pipeline, in nf-core/readmapping/bin/
+    when:
+    task.ext.when == null || task.ext.when
+
+    script: // This script is bundled with the pipeline, in sanger-tol/readmapping/bin/
     """
     check_samplesheet.py \\
         $samplesheet \\
         samplesheet.valid.csv
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        check_samplesheet.py: \$(check_samplesheet.py --version | cut -d' ' -f2)
+        python: \$(python --version | sed 's/Python //g')
```

Comment:
Why not showing the version of the script itself ?

Reply:
I tested the `check_samplesheet.py` script and it work. But I would like to know what you think about the old vs new version. This one seems to use more advanced python syntax than the previous one. I had to look up a lot of the stuff.